src/utilities/export-pdf-helper.ts

```diff
@@ -200,7 +202,9 @@ export default class ExportPDFHelper {
                     })),
                     effectiveScore: grade.effectiveScore,
                     legalScore: grade.legalScore,
-                    srcdoc: src?.renderedHTML ?? 'Could not render this problem.'
+                    srcdoc: src?.renderedHTML ?? 'Could not render this problem.',
+                    startTime: gradeInstanceAttachments?.studentGradeInstance?.createdAt,
+                    submissionTime: gradeInstanceAttachments?.time,
```

Comment:
This seems very wrong

This should be based on the workbook not the attachments

Reply:
Oh the variable is just badly named
```Typescript
const gradeInstanceAttachments = await StudentWorkbook.findOne({
```

I was looking at the real attachments variable
```Typescript
let attachments = gradeInstanceAttachments?.studentGradeInstance?.problemAttachments;
```